tests/includes/class-test-scheduler.php

```diff
@@ -188,7 +188,6 @@ function ( $event ) use ( &$scheduled_time ) {
 
 		// Verify scheduling time.
 		$this->assertGreaterThan( 0, $scheduled_time, 'Event should be scheduled with a future timestamp' );
-		$this->assertGreaterThanOrEqual( time() + 10, $scheduled_time, 'Event should be scheduled at least 10 seconds in the future' );
```

Comment:
These tests are both not overly useful. Maybe we should just check that the event exists.

```php
$this->assertSame( $scheduled_time, wp_next_scheduled( 'activitypub_process_outbox', array( $pending_id ) ) );
```